GitUI/UserControls/FileStatusList.cs

```diff
@@ -967,36 +945,36 @@ private void SelectItems(Func<ListViewItem, bool> predicate)
 
         private void SetDeleteFilterButtonVisibility()
         {
-            DeleteFilterButton.Visible = FilterVisibleInternal && !string.IsNullOrEmpty(FilterComboBox.Text);
+            DeleteFilterButton.Visible = FilterComboBox.Visible && !string.IsNullOrEmpty(FilterComboBox.Text);
         }
 
         private void SetFilterWatermarkLabelVisibility()
         {
-            FilterWatermarkLabel.Visible = FilterVisibleInternal && !FilterComboBox.Focused && string.IsNullOrEmpty(FilterComboBox.Text);
+            FilterWatermarkLabel.Visible = FilterComboBox.Visible && !FilterComboBox.Focused && string.IsNullOrEmpty(FilterComboBox.Text);
         }
 
         private void FileStatusListLoading()
         {
-            LoadingFiles.Visible = true;
+            // Show "Files loading" below the filterbox
             NoFiles.Visible = false;
+            int top = FilterComboBox.Visible ? FilterComboBox.Bottom + FilterComboBox.Margin.Bottom : 0;
+            LoadingFiles.Top = top + 4;
```

Comment:
```suggestion
            LoadingFiles.Top = FilterComboBox.Visible ? FilterComboBox.Bottom + FilterComboBox.Margin.Bottom + LoadingFiles.Margin.Top : 0;
```
Either these 4 pixels are added as `LoadingFiles.Margin.Top` or rather as `Padding` if they should always be added.

Reply:
Must be padding, changed for other 'adjusted' controls too

Reply:
`+ LoadingFiles.Margin.Top` is missing yet - although the value is 0.

Reply:
The `ClearFilter` button is misplaced:

![image](https://github.com/gitextensions/gitextensions/assets/36601201/9827d946-9d94-4e6f-838f-cc2792257479)

Reply:
I had ignored that to limit the line length, now added

Reply:
It was not filling or symmetric before, seem to be more visble now.
Changed to set the height for the button to be the same as for the searchbox
![image](https://github.com/gitextensions/gitextensions/assets/6248932/a2199c74-502c-49b2-a98d-fdd3e00a6f50)